backend-api/src/functions/asyncActiveSession/tokenService/tests/tokenService.test.ts

```diff
@@ -352,7 +355,7 @@ describe("Token Service", () => {
     });
 
     describe("Token signature verification", () => {
-      describe("Given kid from JWT header is not found in JWKS", () => {
+      describe("Given there is an error getting public key from JWKS (e.g. kid from JWT header is not found in JWKS)", () => {
```

Comment:
I've renamed this test as it quite similar to the "Given there is an error retrieving the public key" test above. Any suggestions for a better description here?

Reply:
It's fine line this for now, and it makes sense. We might have to change the tests a bit once we all moving parts come together.